src/index.js

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line no-unused-vars
+const tasks = [
+  {
+    description: 'center a div',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'build a submit button',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'build a form',
+    completed: false,
+    index: 3,
+  },
+];
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Let’s not disable linter rules these are here to help us write a good price of code